src/core/translated.ml

```diff
@@ -7,6 +7,12 @@ type term = {
   translation : (expression, W.t) result;
 }
 
+type check = {
+  txt : string;
+  loc : Location.t;
+  translations : (expression * expression, W.t) result;
```

Comment:
Maybe it could be a good idea to put a few words here stating the role of each of the expressions ?

Reply:
Done